src/runtime/meter-toast-api.js

```diff
@@ -180,7 +180,10 @@ export class MeterToastApi {
           ViewSubscriptionsResponse,
           this.startNativeFlow_.bind(this)
         );
-        if (!this.deps_.callbacks().hasSubscribeRequestCallback()) {
+        if (
+          !this.deps_.callbacks().hasSubscribeRequestCallback() &&
+          !this.deps_.callbacks().hasOffersFlowRequestCallback()
+        ) {
           const errorMessage =
             '[swg.js]: `setOnNativeSubscribeRequest` has not been set ' +
```

Comment:
Looks like this error message needs to be updated.

Reply:
I think this error message is meant to inform the publishers who would like to integrate with this meter toast. For them, the option is still to call `setOnNativeSubscribeRequest`.